app/models/server/raw/VoipRooms.ts

```diff
@@ -0,0 +1,145 @@
+import { Cursor, FilterQuery } from 'mongodb';
+
+import { BaseRaw } from './BaseRaw';
+import { IOmnichannelRoom } from '../../../../definition/IRoom';
+
+type T = IOmnichannelRoom;
+export class VoipRoomsRaw extends BaseRaw<T> {
```

Comment:
That was the initial thought. But after discussing it with the team, we decided that the regression cost would be high. Please find that discussion here. https://open.rocket.chat/group/zfoogyqyrv9i56v23/thread/sNdiXQFQTp2FbCTmG?msg=sNdiXQFQTp2FbCTmG
During this time, I was not aware that there is a way to use same endpoint code using 2 different endpoint paths. 
Furthermore, it felt semantically incorrect to use code path or API path starting with "livechat" because Voip call is not a livechat. So this demanded a refactoring. But we realised that there would have been a significant regression cost. The code is fundamental to the system and any regression would have affected a lot of things. So we decided to duplicate the entire hierarchy of code for Voip. While doing so, I tried to shrink the code for voip in such a way that it has the code that it needs.